### ISSUE TITLE
Link or Headings not working with Hallo RichTextEditor in Internet Explorer

### ISSUE THREAD

**Issue by dominikspiertz** [wagtail/wagtail#3825]
### Issue Summary

We are building a website for Internet Explorer 11 and encountered some errors with the RichTextEditor.

### Steps to Reproduce

Whenever we try to insert a link (doesn't matter if internal or external) or a heading the browser console throws an error:

```
SCRIPT28: Out of stack space
spaceFile: jquery-2.2.1.min.js, Line: 3, Column: 15290
```

After some google-research i figured out, that the provided error occurs in other projects and frameworks too. So what happens is, that internet explorer interprets the jquery to lead it into a infinite recursion call that gets out of stack space soon. 

### Technical details

* Python 3.6.1
* Django 1.11.3
* Wagtail 1.11.1
* Internet Explorer 11 with latest Security Updates on Windows 7 ieVM.

**Comment by thibaudcolas** [comment]
Related: #3400.

**Comment by thibaudcolas** [comment]
Fixed with the new rich text editor, to be released in Wagtail 2.0.
